data/autoyast_xen/sles12sp4PV_PRG.xml

```diff
@@ -1094,36 +1051,13 @@ DefaultPolicy default
     </yesno_messages>
   </report>
   <services-manager>
-    <default_target>graphical</default_target>
+    <default_target>multi-user</default_target>
     <services>
-      <disable config:type="list"/>
+      <disable config:type="list">
+      <service>SuSEfirewall2</service>
+      </disable>
       <enable config:type="list">
-        <service>apparmor</service>
-        <service>btrfsmaintenance-refresh</service>
-        <service>cron</service>
-        <service>display-manager</service>
-        <service>haveged</service>
-        <service>irqbalance</service>
```

Comment:
`irqbalance` we probably still need for the xen test runs, no?

Reply:
Yast have default values for all services. The above services are impotent for OS.  Yast will enable them by default. We don't need to explicitly enable these services in autoyast profile. It's something like: let installer decided what 's best for the OS installation and configuration.